packages/patternfly-4/react-core/src/demos/Toolbar/examples/ComplexToolbarDemo.js

```diff
@@ -69,11 +69,13 @@ class ComplexToolbarDemo extends React.Component {
         position={DropdownPosition.right}
         toggle={<DropdownToggle onToggle={this.onDropDownToggle}>All</DropdownToggle>}
         isOpen={isDropDownOpen}
+        dropdownItems={[
+          <DropdownItem>Item 1</DropdownItem>,
```

Comment:
Since you're touching these lines already, can you add `key`s to the `DropdownItem`s?  The `Dropdown`s are working properly 🎉 but seeing the "unique key prop" warnings in the console.